### PR TITLE
Add Firefox GNOME Theme section

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
Closes https://github.com/EmeraldSnorlax/manjarno/issues/26

### CODE REVIEW

**Comment by EmeraldSnorlax** [comment]
i dont know how to feel about this as well, small bugs do happen and they do result in acessibility changes. this does indeed go against the upstream maintainers explicit wishes though, so im not sure where this fits in. personally, this feels more of a misc. issue but im very much open to hearing other people's considerations

**Comment by EmeraldSnorlax** [comment]
https://stopthemingmy.app/ this might also be apt to mention

**Comment by TheEvilSkeleton** [comment]
Misc works for me, but I'd say stability makes sense as well, since Firefox was practically unusable. I'm okay with any of them. And yes, I agree that Please don’t theme our apps should be mentioned too.

File: src/routes/+page.svelte.md [hunk 1]
```diff
@@ -85,6 +85,32 @@ In their attempt to get Asahi Linux out (and support Apple Silicon) as soon as p
...
+> 
+> **If you are a software distribution maintainer, please do not ship this changes by default to your users unless you made extremely clear that they are using a modified version of Firefox UI.**
+
+The maintainer stated again on Mastodon that they added a disclaimer for that reason:
+
+- https://floss.social/@rafaelmardojai/109532714277650579
+
+While it is not explained in the README, third-party themes are susceptible to accessibility regressions and usability bugs after the app gets updated, in this case Firefox. Third-party themes generally have to update their theme to overwrite the correct and latest interfaces of the app, i.e. the latest version of the app. These third-party themes can take hours, days or even weeks to get updated. It is a complicated step, and it is far from perfect.
+
+Usually, these themes should be left unchanged by default, as Firefox developers have put in the work to improve accessibility in each update, and collectively test the interface for it to be used by the average Firefox user.
```
Comment (EmeraldSnorlax) [inline src/routes/+page.svelte.md:110]: this could be a good place to link to https://stopthemingmy.app/

**Comment by EmeraldSnorlax** [comment]
actually, what do you think of breaking this out into its own heading? manjaro ignoring upstream devs seems to be a repeated trend (e.g. asshi)

File: src/routes/+page.svelte.md [hunk 2]
```diff
@@ -85,6 +85,32 @@ In their attempt to get Asahi Linux out (and support Apple Silicon) as soon as p
 
 Still though, that's besides the main problem. It was [only about 3 days ago this video came out](https://www.youtube.com/watch?v=k0cnMUroMlQ), in which a DE is working for the first time. Not only is it still in a state far from prime-time, but compounded with the tweet above they didn't even bother trying to speak with the devs of the project about its current state. 
 
+### Shipping third-party Firefox theme by default on Manjaro GNOME
```
Comment (EmeraldSnorlax) [inline src/routes/+page.svelte.md:88]: this could be broken out into its own heading of "ignoring upstream" alongside asahi? i'm not sure though

**Comment by TheEvilSkeleton** [comment]
> actually, what do you think of breaking this out into its own heading? manjaro ignoring upstream devs seems to be a repeated trend (e.g. asshi)

I like that. We could use Firefox GNOME Theme as an example, as well as https://dont-ship.it and https://stopthemingmy.app. Besides, they ship a custom GTK theme (and icon theme). I can provide examples along the way.

**Comment by rafaelmardojai** [comment]
Hello, I know you have good intentions here, but I would like to avoid too much drama about this. I hope I can arrive to a constructive discussion with Manjaro, so future users get the best outcome.  (rafaelmardojai/firefox-gnome-theme#528)

Thanks.

**Comment by EmeraldSnorlax** [comment]
i also agree, and i'm glad to see that things are moving in the right direction in terms of discussion -- hopefully the goal for this article is to silently become obsolete as manjaro gets better at running things (even though this started off with not the greatest intentions a few years ago)

i'd also want to get @Chrysostomus's side of things here too; from reading the thread it looks like this is a historical issue that did not fully have the fix propagated to users, in which case this wouldn't be as severe of a misstep and i'd once again be a little reluctant to add here

**Comment by rafaelmardojai** [comment]
> i also agree, and i'm glad to see that things are moving in the right direction in terms of discussion -- hopefully the goal for this article is to silently become obsolete as manjaro gets better at running things (even though this started off with not the greatest intentions from edgy me a few years ago, lol)
> 
> i'd also want to get @Chrysostomus's side of things here too; from reading the thread it looks like this is a historical issue that did not fully have the fix propagated to users, in which case this wouldn't be as severe of a misstep and i'd once again be a little reluctant to add here (at least, not without heavy context)

As far as I know, previously the theme was always applied to users. Now they add it behind a toggle on their greeter application, but the toggle is enabled by default, so is fair to think that many user didn't gave it enough attention.

**Comment by EmeraldSnorlax** [comment]
in that case, i think that still shows them not listening to upstream devs?

**Comment by Chrysostomus** [comment]
I had not noticed the disclaimer in the repo about not shipping it before, but it was brought to my attention recently. Based on the request, it is no longer enabled by default in the next release, and the greeter app will have more information about the theme in the toggle.

**Comment by rafaelmardojai** [comment]
> in that case, i think that still shows them not listening to upstream devs?

I'm not sure about that. This is the first time I openly talked about my will of they stoping doing that.

**Comment by EmeraldSnorlax** [comment]
ah, once again that is my mistake. i don't really see the point of including this pr then, as "small" bugs are out of scope here. feel free to comment with further discussion if wanted

**Pull request closed without merging**